socs/db/suprsync.py

```diff
@@ -208,6 +212,70 @@ def __init__(self, db_path, create_all=True, echo=False):
         if create_all:
             Base.metadata.create_all(self._engine)
 
+    def get_archive_stats(self, session=None):
```

Comment:
I understand why this makes a dict that includes all archive names, but I just want to point out that the only use (outside of tests) is:
```
srfm.get_archive_stats().get(self.archive_name)
```

Which makes me think that it would make sense to have `get_archive_stats()` take an argument for `archive_name` like:
```
srfm.get_archive_stats(self.archive_name)
```